dev/breeze/tests/test_pytest_args_for_test_types.py

```diff
@@ -166,6 +166,7 @@
                 "tests/macros",
                 "tests/notifications",
                 "tests/plugins",
+                "tests/providers",
```

Comment:
This seems to be an artifact from before moving providers to top level folder. Please revert this.